### PR TITLE
GPU: add more fine grained profiling for vertex shader and rasterization

### PR DESCRIPTION
**Pull request by yuriks**

### CODE REVIEW

File: src/video_core/rasterizer.cpp [hunk 1]
```diff
@@ -186,6 +187,8 @@ static int SignedArea (const Math::Vec2<Fix12P4>& vtx1,
     return Math::Cross(vec1, vec2).z;
 };
 
+Common::Profiling::TimingCategory rasterization_category("Rasterization");
```
Comment (neobrain) [inline src/video_core/rasterizer.cpp]: Should this be static?

**Comment by Subv** [comment]
LGTM

File: src/video_core/rasterizer.cpp (hunk 1 above)
Comment (yuriks) [inline src/video_core/rasterizer.cpp]: Good point

**Comment by yuriks** [comment]
Updated

**Comment by neobrain** [comment]
LGTM

**Pull request merged**